### DISCUSSION TITLE
"Create a TypeScript project" options is not presente

### DISCUSSION THREAD

**Discussion by MatteoVeronesi22** [Power-Maverick/DataverseDevTools-VSCode#78] [Q&A] [answered]
I'm testing this interesting  extension but for some reason the "Create a TypeScript project" is not available in the option menù.
What was wrong with the installation?
Thanks in advance

**Comment by Power-Maverick** [comment, marked as answer]
Did you try reopening the folder in VS Code. Some times if you have installed an extension some of the features don't instantly load until you reload it. Also if you have recently updated the extension VS Code needs reload.
